selenium/Fill OTP in MFA.js

```diff
@@ -0,0 +1,21 @@
+// This script can be used to fill in the OTP if it appears right after the authentication.
+// This can only work for the apps who have fixed OTP in MFA
```

Comment:
I think more context would be really useful here.
For example, in this case you have configured browser based auth - this works but then the MFA prompt appears.
It would really help to have a snippet of the HTML of the MFA dialog including the relevant bits like 'flt-glass-pane' and 'flt-semantics-placeholder' - this will help people adapt it to their situations.
This technique will also work if its possible to get the MFA token locally, eg via another command line tool. You dont have to implement that, just maybe mention it?

Reply:
The `flt-glass-pane` thing is only applicable to Flutter apps, I guess we can remove this line as this will probably be useless for all the audience that is not using flutter app and just adds to the complexity of script. What do you say?

Other than that, for HTML do you want me to add a snippet containing OTP input field and submit button?

Reply:
Yep, sounds good. Its just to help people who are have a similar situation but which will obviously be slightly diffferent..